torch/utils/checkpoint.py

```diff
@@ -367,7 +371,9 @@ def inner_unpack(packed):
                     with torch.autograd.graph.saved_tensors_hooks(inner_pack, inner_unpack):
                         _unused = function(*args)
 
-        return storage[x]
+        if x not in storage:
+            raise RuntimeError("Attempted to unpack tensor that has not been packed, please report a bug to PyTorch")
```

Comment:
I'm not sure that sentence makes much sense from the end user point of view? I think "checkpoint" is ok but "packed" is a bit weird.

Reply:
Sure, how about:

`Attempted to retrieve tensor that has not been checkpointed, please report a bug to pytorch`?

Reply:
I think we should mention that this is about retrieving it the tensor twice?
`Retrieving a Tensor saved by autograd multiple times while checkpointing is not currently supported. Please open an issue with details on your use case so that we can prioritize adding this`.